crates/turborepo-ui/src/tui/table.rs

```diff
@@ -124,21 +124,24 @@ impl<'a> StatefulWidget for &'a TaskTable<'a> {
         .column_spacing(0)
         .block(Block::new().borders(Borders::RIGHT))
         .header(
-            vec![format!("Tasks\n{bar}"), " \n─".to_owned()]
-                .into_iter()
-                .map(Cell::from)
-                .collect::<Row>()
-                .height(2),
+            vec![Text::styled(
+                "Tasks\n",
+                Style::default().fg(Color::DarkGray),
+            )]
+            .into_iter()
+            .map(Cell::from)
+            .collect::<Row>()
+            .height(1),
         )
         .footer(
-            vec![
-                format!("{bar}\n{TASK_NAVIGATE_INSTRUCTIONS}\n{HIDE_INSTRUCTIONS}"),
-                format!("─\n "),
-            ]
+            vec![Text::styled(
+                format!("{TASK_NAVIGATE_INSTRUCTIONS}\n{HIDE_INSTRUCTIONS}"),
+                Style::default().fg(Color::DarkGray),
```

Comment:
Maybe use the [`DIM`](https://docs.rs/ratatui/latest/ratatui/style/struct.Modifier.html) modifier instead?
```suggestion
                Style::default().add_modifier(Modifier::DIM),
```

Reply:
Oh, that's fantastic. Didn't know it was there.